sandy_sort_lists.py

```diff
@@ -0,0 +1,94 @@
+import config
```

Comment:
The whole `config.py` thing is quite a hack. We should really be using a `.yaml` file or language-agnostic way to specify configs. It's not hard! I just got lazy. Let's pair on this sometime (or I'll just do it one day and show you pr)